src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -76,7 +67,133 @@ public function elementValidate($element, FormStateInterface $form_state, $form)
       $form_state->setError($element, t('The entity %label is not a group.', $params));
       return;
     }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function form(FieldItemListInterface $items, array &$form, FormStateInterface $form_state, $get_delta = NULL) {
+    $parent_form = parent::form($items, $form, $form_state, $get_delta);
+
+    // Adding the other groups widget.
+    $this->otherGroupsWidget($parent_form['other_groups'], $form_state);
+    return $parent_form;
+  }
+
+  /**
+   * Adding the other groups widget to the form.
+   *
+   * @param $elements
+   *   The widget array.
+   */
+  private function otherGroupsWidget(&$elements, FormStateInterface $form_state) {
+    // @todo check permission.
+    if ($this->fieldDefinition->getTargetEntityTypeId() == 'user') {
+      $description = $this->t('As groups administrator, associate this user with groups you do <em>not</em> belong to.');
+    }
+    else {
+      $description = $this->t('As groups administrator, associate this content with groups you do <em>not</em> belong to.');
+    }
+
+    $elements = [
+      '#type' => 'container',
+      '#tree' => TRUE,
+      '#title' => $this->t('Other widgets'),
+      '#description' => $description,
+      '#prefix' => '<div id="og-group-ref-other-groups">',
+      '#suffix' => '</div>',
+      '#theme' => 'field_multiple_value_form',
+      '#field_name' => $this->fieldDefinition->getName(),
+      '#max_delta' => 1,
+    ];
+
+    $elements['add_more'] = [
```

Comment:
So I think this is because we are having multiple element in two places. I was speaking to @RoySegall a bit about this yesterday, there might be a way to achieve this, but I fear there is probably not. So this might be the best way :)